### PR TITLE
fix(aws): illegal access of private field of AmazonInstance and AmazonServerGroup

### PR DESCRIPTION
**Pull request by pdk27**
This PR fixes the issue - https://github.com/spinnaker/spinnaker/issues/5632 

Previous related commits, in spinnaker:master:
*  https://github.com/spinnaker/clouddriver/pull/4375
* https://github.com/spinnaker/clouddriver/pull/4385

### CODE REVIEW

**Comment by cfieber** [comment]
This was already fixed in #4401 

What version of spinnaker were you running when you encountered spinnaker/spinnaker#5632

File: clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/converters/RebootInstancesAtomicOperationConverter.groovy [hunk 1]
```diff
@@ -47,7 +47,7 @@ class RebootInstancesAtomicOperationConverter extends AbstractAtomicOperationsCr
     try {
       def serverGroups = converted.instanceIds.findResults {
         def instance = amazonInstanceProvider.getInstance(converted.credentials.name, converted.region, it)
-        return instance?.extraAttributes?.serverGroup
```
Comment (cfieber) [inline clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/converters/RebootInstancesAtomicOperationConverter.groovy]: in groovy, `extraAttributes` is equivalent to calling the getter method `getExtraAttributes`

**Comment by cfieber** [comment]
This was the commit with the bug you are seeing:

https://github.com/spinnaker/clouddriver/pull/4375/files#diff-b823bbd08c63189aacc6f1694287d05aR203

This is the fix that is already in place:
https://github.com/spinnaker/clouddriver/pull/4401/files#diff-b823bbd08c63189aacc6f1694287d05aR203

**Comment by pdk27** [comment]
Thanks. Sorry about the churn! I wasn't running the latest.

**Pull request closed without merging**